screens.rpy

```diff
@@ -444,13 +444,14 @@ screen navigation():
         if not persistent.autoload or not main_menu:
 
             if main_menu:
-
-                if persistent.playthrough == 1:
-                    textbutton _("ŔŗñĮ¼»ŧþŀÂŻŕěōì«") action If(persistent.playername, true=Start(), false=Show(screen="name_input", message="Please enter your name", ok_action=Function(FinishEnterName, next_scene_name="start")))
-                else:
+                #Don't need/intend to change the play button like this.
+                #if persistent.playthrough == 1:
+                #    textbutton _("ŔŗñĮ¼»ŧþŀÂŻŕěōì«") action If(persistent.playername, true=Start(), false=Show(screen="name_input", message="Please enter your name", ok_action=Function(FinishEnterName, next_scene_name="start")))
+                #else:
+                if not config.developer:
```

Comment:
I'd appreciate this still being available in developer mode, as we need to test to make sure that the transitions between titlecards work correctly and being able to skip through and rewind is rather useful.

Reply:
Cool, I can change that. Wasn't sure what the intent was. 🙂